src/ansys/dpf/composites/sampling_point_2023r2.py

```diff
@@ -164,7 +164,7 @@ def element_id(self) -> int | None:
         if len(element_scope) > 1:
             raise RuntimeError("The scope of a sampling point can only be one element.")
```

Comment:
```suggestion
            raise RuntimeError("The scope of a sampling point cannot only be one element.")
```
Not part of this PR, but I think there's a typo here.